src/index.html

```diff
@@ -16,7 +28,82 @@
       href="./style.css"
     />
   </head>
+  <header>
+    <img
+      src="./images/logo.png"
+      alt="logo.png"
+    />
+    <nav class="nav">
+      <ul class="nav_list">
+        <li class="nav__item">
+          <a
+            href="#apple"
+            class="nav__link container is-active active"
+          >
+            Apple
+          </a>
+        </li>
+        <li class="nav__item">
+          <a
+            href="#samsung"
+            class="nav__link container"
+          >
+            Samsung
+          </a>
+        </li>
+        <li class="nav__item">
+          <a
+            href="#smartphones"
+            class="nav__link container"
+          >
+            Smartphones
+          </a>
+        </li>
+        <li class="nav__item">
+          <a
+            data-qa="hover"
+            href="#laptops & computers"
```

Comment:
The 'href' attribute value contains spaces, which is not valid. Consider replacing spaces with '%20' or using hyphens or underscores instead.